src/doc/rgw/multisite-reshard.md

```diff
@@ -95,9 +105,14 @@ Reframe in terms of log generations, instead of handling SYNCSTOP events with a
 
 ### Admin APIs
 
-* RGWOp_BILog_List response should include the bucket's highest log generation
+* RGWOp_BILog_List
+    - Add query parameter for 'generation' that defaults to the latest generation.
+    - Add query parameter for 'format-ver' that defaults to 1. If format-ver >= 2, return the results as a struct that includes:
+        * A truncated flag that indicates whether more entries are available.
+        * An optional 'next_generation' field that includes 'num_shards' and generation number of the next log generation.
```

Comment:
A struct 'next_gen' that includes num_shards and gen?

Reply:
that's the info we'll need for the transition, yeah